src/rule.ts

```diff
@@ -12,7 +12,9 @@
  * http://polymer.github.io/PATENTS.txt
  */
 
-import {Document, Warning} from 'polymer-analyzer';
+// TODO(stramel): Remove on drop of Node 6
+// @ts-ignore
```

Comment:
What's up with this annotation?

Reply:
Ah, because it can't name ParsedDocument. I don't think this is a Node 6 thing.

I fixed this locally by importing ParsedDocument and doing this at the bottom of the file. It's ugly but it works:

```typescript
+// Need to have ParsedDocument in exported types for some reason.
+if (false as true) {
+  console.log(null as ParsedDocument | null);
+}
```

Reply:
Ah, I figured it was something like that.

Should I push something like this to it?

Reply:
that works too.

Reply:
@rictic Would you prefer the usage of `ParsedDocument`, the `@ts-ignore` comment directive, or setting the package.json to use `~2.6.2` instead of `^`?

Reply:
Usage of ParsedDocument, definitely. We want to be on 2.7 anyways (it caught a legit typo in a test! `:` instead of `=`), and we don't want to be in the business of ignoring compiler warnings.